src/modules/commander/state_machine_helper.cpp

```diff
@@ -985,6 +993,42 @@ void reset_link_loss_globals(actuator_armed_s &armed, const bool old_failsafe, c
 	}
 }
 
+void set_quadchute_nav_state(vehicle_status_s &status, actuator_armed_s &armed,
+			     const vehicle_status_flags_s &status_flags,
+			     const quadchute_actions_t quadchute_act)
+{
+	switch (quadchute_act) {
+
```

Comment:
```suggestion
```

Reply:
Thank you for your reply @MaEtUgR . I agree that if the position is not valid, it's dangerous to try an RTL. I have thus removed the enforcement of an RTL. I've also modified the early returns and covered the default case of the switch as you proposed. This way, if we have a misconfiguration, we will try to change the nav state to RTL and if it's not possible (i.e. pose not valid) we will not take any action.